commit 43c6b9c03768967d929cc65911af1ff9ecbd40f3
Author: Volker Krause <vkrause@kde.org>
Date:   Sat Jun 22 22:10:02 2024 +0200

    Remove unused KItemModels import
    
    This accidentally added a runtime dependency to KItemModels, which isn't
    covered in Craft's dependency data and thus results in non-starting apps
    if they don't happen to depend on KItemModels by some other way already.

diff --git a/src/dialogs/SearchDialog.qml b/src/dialogs/SearchDialog.qml
index 1cf6ea8a..06da88bf 100644
--- a/src/dialogs/SearchDialog.qml
+++ b/src/dialogs/SearchDialog.qml
@@ -7,7 +7,6 @@ import QtQuick.Controls as QQC2
 import QtQuick.Layouts
 
 import org.kde.kirigami as Kirigami
-import org.kde.kitemmodels
 
 /**
  * A dialog to let's you do a global search accross your applications
